app/controllers/application_controller.rb

```diff
@@ -94,6 +94,10 @@ def current_form
     @current_form ||= Form.find(params[:form_id])
   end
 
+  def current_live_form
+    @current_live_form ||= Form.find_live(params[:form_id])
```

Comment:
What does this return if there's no live form?

Reply:
It would raise a 404 like `current_form`